src/containers/Page/components/Footer.tsx

```diff
@@ -6,187 +6,219 @@
-import { breakpoints, mq, spacing, stackOrder } from "@ndla/core";
-import { FacebookFilled, HelpCircleOutline, Instagram, LinkedIn, Email, Youtube, Launch } from "@ndla/icons/common";
-import { SafeLink } from "@ndla/safelink";
-import { Heading, Text } from "@ndla/typography";
-import { FooterBlock, ZendeskButton } from "@ndla/ui";
+import { HelpCircleOutline, Instagram, LinkedIn, Email, Youtube, Facebook } from "@ndla/icons/common";
+import { Heading, NdlaLogoEn, NdlaLogoNb, Text } from "@ndla/primitives";
+import { SafeLink, SafeLinkIconButton } from "@ndla/safelink";
+import { styled } from "@ndla/styled-system/jsx";
+import { OneColumn, ZendeskButton } from "@ndla/ui";
 import config from "../../../config";
 
-const FooterTextWrapper = styled.div`
-  grid-column: span 2;
-  ${mq.range({ from: breakpoints.tabletWide })} {
-    align-self: flex-end;
-  }
-`;
-
-const ZendeskWrapper = styled.div`
-  position: relative;
-`;
-
-const FooterGrid = styled.div`
-  width: 100%;
-  display: flex;
-  flex-direction: column;
-  gap: ${spacing.normal};
-  ${mq.range({ from: breakpoints.tabletWide })} {
-    display: grid;
-    grid-template-columns: max-content max-content min-content;
-    justify-content: space-between;
-    row-gap: ${spacing.normal};
-  }
-`;
-
-const StyledZendesk = styled(ZendeskButton)`
-  position: absolute;
-  right: ${spacing.large};
-  // Heigth of button is 48px, so this is to center it vertically.
-  top: -${spacing.normal};
-  z-index: ${stackOrder.trigger};
-`;
-
-const StyledHelpCircleOutline = styled(HelpCircleOutline)`
-  width: 20px;
-  height: 20px;
-`;
-
-const StyledLinkBlock = styled.div`
-  display: flex;
-  flex-direction: column;
-  gap: ${spacing.xsmall};
-`;
-
-const LinkList = styled.ul`
-  list-style: none;
-  margin: 0;
-  padding: 0;
-  display: flex;
-  flex-direction: column;
-  gap: ${spacing.xsmall};
-`;
-
-const StyledSafeLink = styled(SafeLink)`
-  color: white;
-  box-shadow: none;
-  text-decoration: underline;
-  text-underline-offset: 5px;
-  &:hover,
-  &:focus,
-  &:active {
-    text-decoration: none;
-  }
-`;
-
-const LinkListElement = styled.li`
-  display: flex;
-  align-items: center;
-  gap: ${spacing.xsmall};
-  padding: 0px;
-  svg {
-    width: 20px;
-    height: 20px;
-  }
-`;
+// TODO: This does not fully adhere to the design. The links switch positions at smaller breakpoints in the design, whereas they remain as-is here.
+// Let's keep things in their natural order for now.
+
+export const FooterBlock = styled("footer", {
+  base: {
+    position: "relative",
+    background: "primary",
+    color: "text.onAction",
+  },
+});
+
+export const StyledOneColumn = styled(OneColumn, {
+  base: {
+    position: "relative",
+    display: "flex",
+    alignItems: "center",
+    justifyContent: "space-between",
+    paddingBlock: "xxlarge",
+    tablet: {
+      paddingBlock: "4xlarge",
+    },
+    desktopDown: {
+      flexDirection: "column-reverse",
```

Comment:
Er dette uu-vennlig da?

Reply:
Ja. Logoen er aria-hidden, så for skjermlesere/tastaturnavigasjon vil det ikke ha noen betydning at ting flytter på seg.